README.md

```diff
@@ -40,6 +40,10 @@ container requires additional information and inputs:
 
 ## Changes
 
+Version 1.0.2 (2024-04-17)
+
+- Add new dependency `inotify-tools`, as `inotifywait` is used by REANA to quickly stop the Kerberos sidecar container when a job finishes its execution.
```

Comment:
Perhaps use plural for better readability?

Add new dependency `inotify-tools` that provides the `inotifywait` utility that is used by REANA 0.95 to quickly stop the Kerberos sidecar containers when the user jobs finish their execution.